src/components/SharedComponents/Button/ButtonInner.tsx

```diff
@@ -88,14 +89,14 @@ const ButtonInner: FC<ButtonInnerProps> = (props) => {
     labelSuffixA11y,
     shouldHideLabel,
     background,
-    variant,
+    variant = "minimal",
```

Comment:
Will this impact other buttons across the app? Might be worth checking with @rightsaidjames

Reply:
Good spot. I've defaulted this to `const DEFAULT_BUTTON_VARIANT: ButtonVariant = "brush";` – which can be found [here](https://github.com/oaknational/Oak-Web-Application/blob/51e8a50fddf428750848048a8e1c477a6a1624bc/src/components/SharedComponents/Button/common.ts#L270), so this now shouldn't affect anything.